unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main/Main.cs

```diff
@@ -87,7 +82,19 @@ protected virtual void SetupEnvironment()
 
         protected virtual HUDContext HUDContextBuilder() { return HUDContextFactory.CreateDefault(); }
 
-        private void Start() { Environment.i.world.sceneController.Start(); }
+        private void Start()
```

Comment:
Is this still called after Awake()?

Reply:
Yes, now that we instantiate new objects on `Awake` we have to wait a bit before they get to initialize and exist, then the `Start` is invoked and everything should work like before

Reply:
👍